### PR TITLE
Adds function for source and target values.

### PR DESCRIPTION
**Pull request by danamouk**
As referenced in issue #14, this change adds a function to view the source and target values before passing them as arguments to the class.

### CODE REVIEW

File: crosswalk.py [hunk 1]
```diff
@@ -25,6 +25,17 @@ def download_data():
     z.extractall(data_directory, members=[i for i in z.namelist()
                  if i in ('CONCEPT.csv', 'CONCEPT_RELATIONSHIP.csv')])
 
+def check_concept_file_source_target_values(concept_filepath):
+     '''
```
Comment (tompollard) [inline crosswalk.py:29]: Looks like this is indented by 5 characters?

Comment (tompollard) [inline crosswalk.py:29]: Probably want double quotes (") for consistency.

File: crosswalk.py [hunk 2]
```diff
@@ -25,6 +25,17 @@ def download_data():
...
+     Checks avaliable source and target vocabulary values from Athena.
```
Comment (tompollard) [inline crosswalk.py:30]: Typo in "avaliable". 

See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html and https://google.github.io/styleguide/pyguide.html for some general advice on styling.

File: crosswalk.py (hunk 1 above)
Comment (danamouk) [reply crosswalk.py]: thanks @tompollard, modified the indent with 4 spaces.

Comment (danamouk) [reply crosswalk.py]: fixed!

File: crosswalk.py (hunk 2 above)
Comment (danamouk) [reply crosswalk.py]: fixed!

**Comment by tompollard** [comment]
Thanks Dana

**Pull request merged**